jekyll/_api/index.html

```diff
@@ -953,9 +953,15 @@ <h3 id='example-call-7'>Example Call</h3><pre class="highlight plaintext"><code>
 <li>the value of path is relative to the project root (the working_directory)</li>
 <li>pretty_path returns the same value as path. It is included in the response for backwards compatibility</li>
 </ul>
-<h2 id='download-an-artifact-file'>Download an artifact file</h2><pre class="highlight plaintext"><code>https://132-55688803-gh.circle-artifacts.com/0//tmp/circle-artifacts.7wgAaIU/file.txt?circle-token=:token
-</code></pre>
-<p>You can download an individual artifact file via the API by appending a query string to its URL. Note that <code>:token</code> is an API token with &#39;view-builds&#39; scope.</p>
+<h2 id='download-an-artifact-file'>Download an artifact file</h2>
```

Comment:
For posterity: this is actually a legacy file that is overwritten by the files in `src-api/source/includes/_artifacts.md ` - which compiles to our v1.1 api during our CI build. No need to revert the file, though!

Reply:
Good to know!